.github/actions/update-github-issue-and-pull-request/Update-GitHubIssueAndPullRequest.ps1

```diff
@@ -41,8 +41,19 @@ elseif ($Body -NotLike "*``[$issueKey``]``($jiraBrowseUrl$issuekey``)*")
 }
 
 $issueQuery = "$issueKey in:title"
-$output = gh issue list --search $issueQuery --repo $GitHubRepository
-$issueItem = ($output | Select-Object -First 1)
+$issueItem = $null
+$issuesEnabled = (gh repo view $GitHubRepository --json hasIssuesEnabled | ConvertFrom-Json).hasIssuesEnabled
+
+if ($issuesEnabled)
+{
+    $output = gh issue list --search $issueQuery --repo $GitHubRepository
+    $issueItem = ($output | Select-Object -First 1)
+}
+else
```

Comment:
We don't need the else branch. If issues are disabled, then `$issueItem` remains `$null`, but we still need to continue execution, because the PR can still be updated with the link to the Jira issue (that's why it was reopened). Just add what I commented to the issue.